### PR TITLE
449 use the python installer for linux installation

### PR DESCRIPTION
**Pull request by thomas-senechal**

### CODE REVIEW

**Comment by gregLibert** [comment]
@SlnPons There is no replacement of the existing script + an update of the installation procedure because I would like to start testing it before.
And by testing I mean checking that it passes the tests that I asked @Pierregvx to do.

File: .github/workflows/build.yml
Comment (gregLibert) [reply .github/workflows/build.yml:80]: okidoki. Done

File: scripts/installer.py
Comment (gregLibert) [reply scripts/installer.py:198]: my bad, sorry.

**Comment by thomas-senechal** [comment]
Didn't tested it and I can't approve it since I'm the one that made this PR. 

You have my approval.

File: scripts/installer.py
Comment (Thykof) [inline scripts/installer.py:151]: warning, `file` parameter is not used.

don't we have a linter that can spot that before pushing or in the CI?

File: scripts/linux_installer.py
Comment (Thykof) [inline scripts/linux_installer.py:79]: I think here you can use the methods defined in base classe Installer

File: scripts/linux_installer.py
Comment (Thykof) [inline scripts/linux_installer.py:22]: if you want, you can name this unsused variable `_`. but I understand if you want to be clear about what this variable is and that we deliberately don't want to use it. (this remark apply for some other cases in the code)

```suggestion
        stdout, _ = self.executeCommand("grep '^dns=' /etc/NetworkManager/NetworkManager.conf", True, allow_failure=True)
```

File: scripts/linux_installer.py
Comment (Thykof) [inline scripts/linux_installer.py:61]: in that case maybe we should make sure that dndmasq is installed?

```suggestion
                self.executeCommand("sudo apt-get install -y dnsmasq", True)
                self.configureNetworkManager()
```

**Review by Thykof — CHANGES_REQUESTED** [review]
Some python and logical remarks, some are optional :) about installation I can't really say if it's OK

File: scripts/linux_installer.py
Comment (gregLibert) [reply scripts/linux_installer.py:79]: No, the command needs sudo...

**Review by Thykof — APPROVED** [review]

**Comment by SlnPons** [comment]
@Pierregvx can you test this issue by making sure that it passes the end-to-end test that you wrote for linux please?

**Pull request merged**
